### PR TITLE
Fix #6446: Changing Brand / Interaction Color

### LINKED ISSUE #6446: Changing Brand / Interaction Color
## Problem Description 

Design team wants to change orange traditional brand/interaction color. and we have started implementing a blue color for some new features. Our design team is moving forward with blue and so we want to go and update all the orange we have in the code to blue.

## Feature Overview 

- Switching the orange to the purple, this will be a more manual process finding a lot of the places we use braveOrange and swapping it with the blurple

- Updating the actual blurple can be done in the asset catalog in the DesignSystem module/folder 

- Additionally include some parts of the app which are missing accent color on checkmark and toggle.

### PR DESCRIPTION
**Pull request by soner-yuksel**
And adding the tint accent color thorough out the app

## Summary of Changes

This pull request fixes #6446

- Switching the orange to the purple, finding places we use braveOrange and swapping it with the blurple

- Updating the actual blurple can be done in the asset catalog in the DesignSystem module/folder

- Additionally include some parts of the app which are missing accent color on checkmark and toggle.

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`
- [x] New or updated UI has been tested across:
  - [x] Light & dark mode
  - [x] Different size classes (iPhone, landscape, iPad)
  - [x] Different dynamic type sizes

## Reviewer Checklist:

- [ ] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [ ] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [ ] Adequate unit test coverage exists to prevent regressions.
- [ ] Adequate test plan exists for QA to validate (if applicable).
- [ ] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Review by StephenHeaps — APPROVED** [review]
Wallet is updated, thanks 👍. I'd wait for approval from others in @brave/ios to merge.

**Review by kylehickinson — APPROVED** [review]
So purple 😵‍💫

**Pull request merged**
